client/my-sites/comments/comment/style.scss

```diff
@@ -6,14 +6,18 @@
 	font-size: 14px;
 	margin: 0 auto;
 	padding: 0;
-	transition: margin 0.15s linear;
 
 	.accessible-focus &:focus {
 		box-shadow: 0 0 0 1px $blue-medium, 0 0 0 3px $blue-light;
 		z-index: z-index('root', '.comment-detail.card.accessible-focus:focus');
 	}
 }
 
+// `transition` here is applied with less specificity to avoid overwriting ReactCSSTransitionGroup's animation.
+.comment {
```

Comment:
I double checked that we're not colliding with reader components, but heads up if we want to rename comment-list to comments. The later is already taken by the comments block.

Reply:
Yup, I made sure to check if anything else was using the `.comment` class before committing to the `Comment` name. 🙂